layers/+lang/plantuml/packages.el

```diff
@@ -0,0 +1,27 @@
+;;; packages.el --- plantuml layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Robert O'Connor <robby.oconnor@gmail.com>
+;; URL: https://github.com/robbyoconnor
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;;; Code:
+
+(defconst plantuml-packages
+  '(puml-mode))
+
+(defun plantuml/init-puml-mode ()
+  (use-package puml-mode
+    :defer t
+    :mode ("\\.puml" . puml-mode) ("\\.plantuml" . puml-mode)
```

Comment:
Also are these extensions used? It [seems](https://github.com/skuro/puml-mode/blob/master/puml-mode.el#L278) the official one is `.pum`.